SPECS/ca-certificates/ca-certificates.spec

```diff
@@ -44,7 +44,7 @@ Name:           ca-certificates
 
 # When updating, "Epoch, "Version", AND "Release" tags must be updated in the "prebuilt-ca-certificates*" packages as well.
 Epoch:          1
-Version:        %{azl}.0.0
+Version:        3.0.0
```

Comment:
I think these changes got add accidentally after a recent fix in `3.0-dev`. Maybe a merge/rebase of the latest `3.0-dev` will get rid of these.